spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -311,6 +364,153 @@ def _measure_axial_damage_ratio(self, im_data, p_lst, idx):
+        for sagittal_slice in sagittal_lesion_slices:
+            # Get df for the selected sagittal slice
+            df_temp = tissue_bridges_df[tissue_bridges_df['sagittal_slice'] == sagittal_slice]
+            # Get the axial slices corresponding to the minimum bridge widths
+            min_dorsal_bridge_width_slice = df_temp.loc[df_temp['dorsal_bridge_width'].idxmin(), 'axial_slice']
+            min_ventral_bridge_width_slice = df_temp.loc[df_temp['ventral_bridge_width'].idxmin(), 'axial_slice']
+
+            # Add a new column with value True to tissue_bridges_df; this information is needed for plotting
+            tissue_bridges_df.loc[(tissue_bridges_df['sagittal_slice'] == sagittal_slice) &
+                                  (tissue_bridges_df['axial_slice'] ==
+                                   min_dorsal_bridge_width_slice), 'min_dorsal_bridge_axial_slice'] = True
+            tissue_bridges_df.loc[(tissue_bridges_df['sagittal_slice'] == sagittal_slice) &
+                                  (tissue_bridges_df['axial_slice'] ==
+                                   min_ventral_bridge_width_slice), 'min_ventral_bridge_axial_slice'] = True
+            # Replace NaN with False
+            tissue_bridges_df.fillna(False, inplace=True)
+
+            # Get the width of the tissue bridges in mm
+            # NOTE: the orientation is RPI (because we reoriented the image to RPI using orient2rpi()); therefore
+            # p_lst[0] is the pixel size in the R-L direction, p_lst[1] is the pixel size in the A-P direction, and
+            # p_lst[2] is the pixel size in the S-I direction.
+            # Since we are computing dorsal and ventral tissue bridges, we use p_lst[1] (A-P direction)
+            # NOTE: we use np.cos(self.angles[SLICE]) to correct for the angle of the spinal cord with respect to the
+            # slice
+            dorsal_bridge_width_mm = (float(df_temp[df_temp['axial_slice'] == min_dorsal_bridge_width_slice]
+                                            ['dorsal_bridge_width'] * p_lst[1]) *
+                                      np.cos(self.angles[min_dorsal_bridge_width_slice]))
+            ventral_bridge_width_mm = (float(df_temp[df_temp['axial_slice'] == min_ventral_bridge_width_slice]
+                                             ['ventral_bridge_width'] * p_lst[1]) *
+                                       np.cos(self.angles[min_ventral_bridge_width_slice]))
+            total_bridge_width_mm = dorsal_bridge_width_mm + ventral_bridge_width_mm
+
+            # Save the minimum tissue bridges
+            self.measure_pd.loc[idx, f'slice_{str(sagittal_slice)}_dorsal_bridge_width [mm]'] = dorsal_bridge_width_mm
+            self.measure_pd.loc[idx, f'slice_{str(sagittal_slice)}_ventral_bridge_width [mm]'] = ventral_bridge_width_mm
+            self.measure_pd.loc[idx, f'slice_{str(sagittal_slice)}_total_bridge_width [mm]'] = total_bridge_width_mm
```

Comment:
I think these explicit calls to `str()` are redundant?
```suggestion
            self.measure_pd.loc[idx, f'slice_{sagittal_slice}_dorsal_bridge_width [mm]'] = dorsal_bridge_width_mm
            self.measure_pd.loc[idx, f'slice_{sagittal_slice}_ventral_bridge_width [mm]'] = ventral_bridge_width_mm
            self.measure_pd.loc[idx, f'slice_{sagittal_slice}_total_bridge_width [mm]'] = total_bridge_width_mm
```

Reply:
you are right, thanks! `str()` removed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4489/commits/1f50b3b05f6c90351857da8ef0df69c087169719